data/core/doc/init.lua

```diff
@@ -499,6 +544,34 @@ function Doc:indent_text(unindent, line1, col1, line2, col2)
   return line1, col1 + #text, line1, col1 + #text
 end
 
+function Doc:update_max_line_len_range(start_line, end_line)
+  local line_numbers = self.long_lines.line_numbers
+  local max_length = self.long_lines.length
+  end_line = math.min(end_line, #self.lines)
+
+  for line=start_line,end_line do
+    local line_len = string.len(self.lines[line])
+    if line_len > max_length then
+      max_length = line_len
+      line_numbers = { [line] = true }
+    elseif line_len == max_length then
+      line_numbers[line] = true
+    else
+      if line_numbers[line] then line_numbers[line] = nil end
+    end
+  end
+
+  if common.get_table_size(line_numbers) == 0 then
```

Comment:
The function `common.get_table_size` should be removed. To check if a table has no elements the condition `next(t) == nil` can be used.

Reply:
Well, that's handy. I didn't know about it, I still need to research lua more.